src/components/marker-table/ContextMenu.js

```diff
@@ -101,6 +101,30 @@ class MarkersContextMenu extends PureComponent<Props> {
     });
   };
 
+  setRangeByDuration = () => {
+    const { selectedMarker, markers, updatePreviewSelection } = this.props;
+
+    if (selectedMarker === null) {
+      return;
+    }
+
+    const marker = markers[selectedMarker];
+    if (this._isZeroDurationMarker(marker)) {
+      return;
+    }
+
+    updatePreviewSelection({
+      hasSelection: true,
+      isModifying: false,
+      selectionStart: marker.start,
+      selectionEnd: marker.start + marker.dur,
+    });
+  };
+
+  _isZeroDurationMarker(marker: any): boolean {
```

Comment:
We try to not use `any` unless we're totally forced to do so, because this makes Flow completely skip the check. In this case I think you can use `?Marker`.

Reply:
I rebased and force pushed the type check. Thanks!